docs/source/api_ref_models.rst

```diff
@@ -133,6 +133,29 @@ To download the CodeLlama-7B model:
     code_llama2.lora_code_llama2_70b
     code_llama2.qlora_code_llama2_70b
 
+qwen-2
+------
+
+Models from the `Qwen2 small model family <https://huggingface.co/collections/Qwen/qwen2-6659360b33528ced941e557f>`_.
+
+To download the Qwen2 1.5B model, for example:
+
+.. code-block:: bash
+
+    tune download Qwen/Qwen2-0.5B-Instruct --output-dir /tmp/Qwen2-0.5B-Instruct --ignore-patterns None
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    qwen2.qwen2_7b
+    qwen2.qwen2_0_5b
+    qwen2.qwen2_1_5b
+    qwen2.lora_qwen2_7b
+    qwen2.lora_qwen2_0_5b
+    qwen2.lora_qwen2_1_5b
+    qwen2.qwen2_tokenizer
+    qwen2.Qwen2Tokenizer
```

Comment:
think you need to add this to the init

Reply:
should be a doozy, pal

Reply:
you got this boss

Reply:
![image](https://github.com/user-attachments/assets/fd10a56f-9b69-4106-97ce-4fe5a341c412)